### PR TITLE
chore: Upgrade golang.org/x/crypto 

### PR DESCRIPTION
**Pull request by spolti**
chore:	address the following CVE:
 	- [CVE-2023-48795](https://www.cve.org/CVERecord?id=CVE-2023-48795): golang.org/x/crypto Authentication Bypass by Capture-replay

### CODE REVIEW

**Comment by spolti** [comment]
@rafvasq hey, if you could take a look on this one as well :)

File: go.mod [hunk 1]
```diff
@@ -37,9 +37,12 @@ require (
...
-	// fixes pkg/mod/github.com/golang/glog@v1.1.0/internal/logsink/logsink.go:123:41: undeclared name: any (requires version go1.18 or later)
-	// remove when updating to go lang 1.19
-	github.com/golang/glog => github.com/golang/glog v1.0.0
+	// The x/crypto is pulled from golang.org/x/net@v0.10.0
+	// Fixes CVE-2023-48795 - golang.org/x/crypto Authentication Bypass by Capture-replay
+	golang.org/x/crypto => golang.org/x/crypto v0.17.0
 	golang.org/x/net => golang.org/x/net v0.17.0
+	// remove when upgrade to controller-runtime 0.15.xor apimachinery to 0.27.x
+	// Fixes github.com/elazarl/goproxy Denial of Service (DoS)
+	// This dependency was remove from apimachinery 0.27.0
```
Comment (rafvasq) [inline go.mod:46]: Just a suggestion, I think it's a bit more succinct for whoever handles it next.

```suggestion
	
	// Fixes github.com/elazarl/goproxy Denial of Service (DoS), the dependency was removed in v0.27.0
	// Remove below when upgrading to controller-runtime 0.15.x or apimachinery to 0.27.x
```

**Review by rafvasq — COMMENTED** [review]
Just a suggestion to clean up the dependency comment.

File: go.mod (hunk 1 above)
Comment (spolti) [reply go.mod]: cool.
applied, thx.

**Review by rafvasq — APPROVED** [review]
LGTM

**Pull request merged**